basis/concurrency/distributed/distributed-docs.factor

```diff
@@ -39,9 +41,9 @@ $nl
 $nl
 "It is also possible to use " { $link send-synchronous } " to receive a "
 "response to a distributed message. When an instance of " { $link thread } " "
-"is marshalled it is converted into an instance of " { $link remote-thread }
+"is marshalled, it is converted into an instance of " { $link remote-thread }
 ". The receiver of this can use it as the target of a " { $link send }
-" or " { $link reply } " call." ;
+", " { $link send-synchronous } " or " { $link reply-synchronous } " call." ;
```

Comment:
Why did you remove ``send`` and ``reply`` in favor of the synchronous versions?  Maybe we should mention both versions.